.github/workflows/release-nightly.yml

```diff
@@ -71,5 +71,5 @@ jobs:
           goos: ${{ matrix.goos }}
           goarch: ${{ matrix.goarch }}
           ldflags: -s -w -X github.com/ignite/cli/ignite/version.Version=nightly
-          goversion: "./go.mod"
+          goversion: "1.21.2"
```

Comment:
According to `wangyoucao577/go-release-action` yes, we could just use "latest" as value but I didn't see it handled in the code that creates the download URL. In any case even if it works I was not sure to use it because I don't know if it would be ok to use a new minor version for `nightly` before we bump the other workflows and CLI's Go mod.

I have no opinion though so we can change it to "latest", I was just being cautious.

Reply:
We Go backward compatibility guarantees, our code should not break if we build with later versions of Go.
Hard-coding it, means we need to bump every month of so.